tests/conftest.py

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+
+@pytest.fixture(scope='session')
+def capabilities(capabilities):
+    capabilities.setdefault('tags', []).append('socorro')
+    return capabilities
+
+
+@pytest.fixture
+def existing_user(variables):
+    return variables['users']['default']
+
+
+@pytest.fixture
+def selenium(selenium):
+    selenium.implicitly_wait(10)
+    selenium.maximize_window()
```

Comment:
As previously mentioned, maximise is going to be different depending on the resolution of the screen where the tests are running. If there's a preferred window size it's better to specify that, so that it's more consistent between users. This matters even more when there are floating UI elements that cause scrolling. Feel free to leave this in, I'm only warning and wanting to make sure the implications are understood. This is a bad practice imho.